mopidy_spotify/web.py

```diff
@@ -75,7 +75,14 @@ def _refresh_token(self):
             logger.debug('Token scopes: %s', data['scope'])
 
     def get(self, url, **kwargs):
-        if self._should_refresh_token():
-            self._refresh_token()
-        kwargs.setdefault('headers', {}).update(self._headers)
-        return self._request('GET', url, **kwargs)
+        result = {}
+        try:
+            if self._should_refresh_token():
+                self._refresh_token()
+            kwargs.setdefault('headers', {}).update(self._headers)
+            response = self._request('GET', url, **kwargs)
+            result = self._decode(response)
+        except Error as e:
```

Comment:
We should probably catch a more specific error? Also do we want to log this just as debug?